core/src/main/java/org/apache/brooklyn/core/mgmt/rebind/dto/BasicEntityMemento.java

```diff
@@ -139,17 +142,28 @@ protected BasicEntityMemento(Builder builder) {
         
         effectors = toPersistedList(builder.effectors);
         
+        allConfigKeys = builder.configKeys;
         configByKey = builder.config;
         configUnmatched = builder.configUnmatched;
         attributesByKey = builder.attributes;
         
+        staticConfigKeys = getStaticConfigKeys();
+        staticSensorKeys = getStaticSensorKeys();
+        
         if (configByKey!=null) {
             configKeys = Maps.newLinkedHashMap();
             config = Maps.newLinkedHashMap();
+            
+            for (ConfigKey<?> key : allConfigKeys) {
+                if (!key.equals(staticConfigKeys.get(key.getName()))) {
+                    configKeys.put(key.getName(), key);
+                }
+            }
```

Comment:
This will remove entries which override the default value. I'd use reference comparison here.